distributed/comm/tests/test_ucx.py

```diff
@@ -234,9 +227,7 @@ def test_ucx_deserialize():
     "n",
     [
         int(1e9),
-        pytest.param(
-            int(2.5e9), marks=[pytest.mark.xfail(reason="integer type in ucx-py")]
-        ),
+        int(2.5e9),
```

Comment:
Was marked as `xfail`, but I haven't seen any failures. So have just allowed it to run.

This whole test is still marked as slow, but doesn't seem to be that slow. Maybe we should just run it by default?

Reply:
Added PR ( https://github.com/rapidsai/ucx-py/pull/474 ) to start running the slow tests as part of UCX-Py's CI runs.